task/power/src/main.rs

```diff
@@ -140,6 +145,22 @@ impl Device {
         };
         Ok(r)
     }
+
+    fn pmbus_read(
+        &self,
+        op: task_power_api::Operation,
+    ) -> Result<PmbusValue, ResponseCode> {
+        let v = match &self {
+            Device::Mwocp68(dev) => dev.pmbus_read(op)?,
+            Device::Bmr491(_)
+            | Device::Raa229618(_)
+            | Device::Isl68224(_)
+            | Device::Tps546B24A(_)
+            | Device::Adm1272(_)
+            | Device::Max5970(_) => return Err(ResponseCode::NoDevice),
```

Comment:
Do we want to return a distinct error code here to indicate that the operation is not supported rather than the device is not present?

Reply:
Good call - added in 4ec47222